.github/workflows/ci-actions.yml

```diff
@@ -517,6 +517,7 @@ jobs:
               resteasy-mutiny
               vertx
               vertx-http
+              vertx-web
```

Comment:
Does the timeout perhaps need to be increased or do we already have some headroom?

Reply:
Added 5 extra mins.